modules/managed-resources.adoc

```diff
@@ -0,0 +1,70 @@
+:_mod-docs-content-type: CONCEPT
+//[id="sd-managed-resources"]
+[id="sd-redhat-managed-resources_{context}"]
+= Red{nbsp}Hat managed resources
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: sd-managed-resources
+
+toc::[]
+
+[id="sd-managed-resources-overview_{context}"]
+== Overview
+
+The following covers all {product-title} resources that are managed or protected by the Service Reliability Engineering Platform (SRE-P) Team. Customers should not attempt to modify these resources because doing so can lead to cluster instability.
```

Comment:
```suggestion
The following covers all {product-title} resources that are managed or protected by the Service Reliability Engineering Platform (SRE-P) Team. Customers should not try to change these resources because doing so can lead to cluster instability.
```